content/tutorials/token-authentication.textile

```diff
@@ -797,17 +804,23 @@ blang[php].
   "See this step in Github":https://github.com/ably/tutorials/commit/f949bcf
 
 blang[swift].
-  First we need to set up an Xcode project that will serve as our client.
-  To build your own Xcode Project in Swift visit "Apple developer website":https://developer.apple.com/library/content/documentation/IDEs/Conceptual/AppStoreDistributionTutorial/Setup/Setup.html and get familiar with steps necessary to setup your own application.
-  When you setup your application delete the default @ViewController.swift@ add new File -> New -> File... and choose @Cocoa Touch Class@.
+  Firstly you'll need to set up an Xcode project that will serve as your client.
```

Comment:
"you'll" => "you"

Reply:
They will be setting it up, thus you'll, no?

Reply:
Sorry @tomczoink probably influenced by previous comments :) 'You'll' sounds fine here!